frontend/src/APIClients/BaseAPIClient.ts

```diff
@@ -126,9 +127,10 @@ const errorLink = onError(({ graphQLErrors, networkError }) => {
       /* eslint-enable no-console */
     });
   }
-  // eslint-disable-next-line no-console
-  if (networkError) console.log("[Network error]", networkError);
-  if (networkError?.message === "Error: Failed to refresh access token!") {
+  // if (networkError?.toString() === "Error: Failed to refresh access token!") {
+  //   window.location.href = "/logout";
+  // }
+  if (networkError) {
```

Comment:
This seems a little risky since I'd imagine this would also sign people out if their internet briefly disconnected. Is there a particular reason to change the logic here? (i.e., was it not working before?)